docs/enforcement/inline.asciidoc

```diff
@@ -0,0 +1,255 @@
+
+* PacketFence provides its own DHCP service. It will take care of IP address distribution in our Inline network. PacketFence will not provide DHCP services on the management network - this is the responsibility of your own infrastructure.
+* PacketFence provides its own DNS service. However for the Inline version, we need to provide the DNS of your infrastructure.
+
+From *Configuration->Network Configuration->Interfaces*, click on the `ens192` logical name. Provide the following information:
+
+    IP Address: 192.168.2.1
+    Netmask: 255.255.255.0
+    Type: Inline Layer 2
+    Additionnal listening daemon(s): portal
+    DNS Servers: 10.0.0.10
+
+Click on 'Save' and toggle the new interface to 'On'.
+
+Once done, your PacketFence server should have the following network layout:
+
+Please refer to the following table for IP and subnet information :
+
+[options="header",cols="1,2,2,2,3",grid="rows"]
+|===================================================================================
+|Network Card    |Name          |Subnet            |Gateway        |PacketFence Address
+|ens160          |Management    |172.20.100.0/24   |192.168.1.1    |172.20.100.2
+|ens192          |Inline        |192.168.2.0/24    |192.168.2.1    |192.168.2.1  
+|===================================================================================
+
+Finally, from *Status->Services*, restart the `haproxy` `httpd.portal`, `dhcpd`, `iptables`, `pfdhcplistener`, `pfdns` services.
+
+
+Network Devices
+~~~~~~~~~~~~~~~
+
+In an inline configuration, the required configurations for network devices (desktops, tablets, printers, etc.) will be to make sure they can all communicate with PacketFence. In other words for a switch you will need to configure every ports on which devices will be connected using the access mode with all of them in the same inline network. Access point will be connected as device to be in the inline subnetwork. 
+
+Example with a Cisco switch:
+
+You should be in mode '#conf-t' if not execute 'configuration terminal' in your CLI.
+----
+# interface range [port-range]
+# switchport mode access vlan 1
+# no shutdown
+# interface [packetfence_ens192]
+# switchport mode access vlan 1
+# no shutdown
+# end
+# copy running-configuration startup-configuration
+----
+
+Now you can connect any devices that you want to be in the inline network in any of the port you have just configured.
+
+Adding Connection Profile for Inline
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next thing we do is to add a new connection profile - for devices coming from the inline network. We want to show users the captive portal with our Null authentication sources.
+
+From *Configuration->Policies and Access Control->Connection Profiles", click on 'Add Profile'. Provide the following information:
+
+ * Profile Name: inline
+ * Filters: If *net* Network 192.168.2.0/24
+ * Sources: null-source
+
+Then click on Save.
+
+Advanced Inline Topics
+~~~~~~~~~~~~~~~~~~~~~~
+
+Traffic Shaping
```

Comment:
Not sure this section is still accurate with the work @fdurand and @jrouzierinverse did for the inline TC